.circleci/config.yml

```diff
@@ -794,7 +794,7 @@ workflows:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - prepare-next-version:
-          xcode_version: '14.1.0'
+          xcode_version: '14.2.0'
```

Comment:
Hmm I think we could probably run some of these jobs in non-mac machines... But that doesn't belong to this PR.

Reply:
Yeah there's lots we could do to speed up some of these. Which ones were you thinking specifically?

Reply:
Oh like `prepare-next-version`, yeah. I think it's so we can have some basic stuff installed, but I'm sure they're lighter images we could use.

Reply:
Yeah, I was thinking on any like this, where it's just running some fastlane scripts.